### PR TITLE
Add Views

### PR DESCRIPTION
**Pull request by yasinabdmahmood**
### Project changes
- Implemented the design from the sneak peek wireframes.
    
    
image
image
image
image

- Used methods that you have created in the [Blog App - processing data in models](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Processing_data_model_project.md).
- Added new records to all the table in database

### CODE REVIEW

File: app/views/users/show.html.erb [hunk 1]
```diff
@@ -1 +1,35 @@
...
+          <img 
+          class="face" 
+          src=<%= @user.Photo %>
+          alt="face" />
+        </div>
+        <div class="details">
+            <h1><%= @user.Name %></h1>
+            <h3>Number of posts : <%= @user.PostsCounter %></h3>
+        </div>
+    </div>
```
Comment (FrederickMih) [inline app/views/users/show.html.erb:15]: - [**Optional**] You are repeating this piece of code in most of your views pages. Let's try to keep the codebase clean and DRY by using [Rails partials](https://guides.rubyonrails.org/layouts_and_rendering.html).

File: app/views/posts/index.html.erb [hunk 2]
```diff
@@ -1 +1,39 @@
...
+      <a href="/users/<%= @user.id %>/posts/<%= post.id %>">
+      <div>
+          <div class="post">
+            <h3><%= post.Title %></h3>
+            <p><%= post.Text %></p>
+            
+          </div>
+          
+            <div class="comment-details">
+            <% post.comments.each do |comment| %>
```
Comment (FrederickMih) [inline app/views/posts/index.html.erb]: - [ ] Hi @yasinabdmahmood, let's retrieve and render the five most recent comments of each post. It is mentioned in the requirements. The piece of code below does the job I think.

```ruby
 <% post.retrieve_recent_comments.each do |comment| %>
```

- Only post show page should display all these comments

**Review by FrederickMih — CHANGES_REQUESTED** [review]
## Hi :raising_hand: @yasinabdmahmood, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!




## Highlights

- The design  of all the pages matches the given wireframes :heavy_check_mark: 
- Descriptive README.md file :+1: 
- Great commit messages :+1: 
- PR is descriptive :+1: 
- No Linter error :heavy_check_mark: 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/views/posts/index.html.erb (hunk 2 above)
Comment (Whoistolu) [reply app/views/posts/index.html.erb]: Well resolved 👍🏽

**Review by Whoistolu — APPROVED** [review]
### Hi @yasinabdmahmood 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

# STATUS: APPROVED 🟢
**Congratulations! 🎉** 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**